nsqadmin/static/js/lib/handlebars_helpers.js

```diff
@@ -61,17 +61,24 @@ var genColorList = function(typ, key) {
     return 'blue';
 };
 
+// sanitizeGraphiteKey removes special characters from a graphite key
+// this matches behavior of bitly/statsdaemon
+// https://github.com/bitly/statsdaemon/blob/fc46d9cfe29b674a0c8abc723afaa9370430cdcd/statsdaemon.go#L64-L88
+var sanitizeGraphiteKey = function(s) {
+    return s.replaceAll(/[ ]/g, '_').replaceAll('/[/]/g', '-').replaceAll(/[^a-zA-Z0-9-_.]/g, '');
```

Comment:
the quotes around `'/[/]/g'` make it not work:
![js_replace_ex](https://user-images.githubusercontent.com/649835/129494758-2d00ad4d-b48a-4ef5-8fdd-450264a43645.png)

That's because the first argument to `replaceAll()` can either be a regex obj or a string (and the quotes make it a string, not a regex). But this suggests a way to simplify these calls:

```js
s.replaceAll(' ', '_').replaceAll('/', '-')
```

![replace_simple_ex](https://user-images.githubusercontent.com/649835/129494814-449ca32c-f1fe-4fdc-9c1a-86919317a0cf.png)